e2e/province.e2e.js

```diff
@@ -64,3 +78,55 @@ describe('Test province flow', () => {
     await device.takeScreenshot('DiagnosedShare');
   });
 });
+
+describe('Test region based screens', () => {
+  it('displays right no code screen for ON and AB', async () => {
+    await changeRegion('ON');
+    await element(by.id('tapPromptCollapsed')).tap();
+    await expect(element(by.id('getCodeButton'))).toBeVisible();
+    await element(by.id('getCodeButton')).tap();
+    await device.takeScreenshot('noCodeON');
+    await expect(element(by.id('noCodeHeader'))).toBeVisible();
+    await expect(element(by.id('noCodeCTA'))).toBeVisible();
+    await element(by.id('toolbarCloseButton')).tap();
+    await element(by.id('bottom-sheet-close')).tap();
+    await changeRegion('AB');
+    await element(by.id('tapPromptCollapsed')).tap();
+    await expect(element(by.id('getCodeButton'))).toBeVisible();
+    await element(by.id('getCodeButton')).tap();
+    await device.takeScreenshot('noCodeAB');
+    await expect(element(by.id('noCodeHeader'))).toBeVisible();
+    await expect(element(by.id('noCodeCTA'))).toBeNotVisible();
+    await element(by.id('toolbarCloseButton')).tap();
+    await element(by.id('bottom-sheet-close')).tap();
+  });
```

Comment:
I think this test will pass even if AB and ON display the same content on the no code screen. Could you check for either different text, or that the button only exists on the ON screen?

Reply:
@smcmurtry I fixed up the Android tests and I am using `await expect(element(by.id('noCodeCTA'))).not.toBeVisible();` to determine if the CTA button is visible on the No Code screen, is this sufficient for testing?

Reply:
Yes that is great!